.github/workflows/build-and-test-multiplatform.yml

```
@@ -0,0 +1,350 @@
+      with:
+        name: results-linux-gcc-x64
+        path: build/results
+        retention-days: 7
+
+  summarize:
+    name: Summarize and Send PR Message
+    runs-on: ubuntu-24.04
+    needs: [test-windows-x64, test-macos-x64, test-linux-clang-x64, test-linux-gcc-x64]
+
+    steps:
+    - name: Checkout repository
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)